jest.config.cjs

```diff
@@ -6,7 +6,7 @@ const config = {
   coverageThreshold: {
     global: {
       statements: 59.23,
-      branches: 70.32,
+      branches: 70.1,
```

Comment:
Generally, I would say that when adding new features, test coverage should not go down. I would be curious to see in your coverage report what parts of the code you added were not tested by the cases I laid out. That would be on me, not you, but as a dev it's a good thing to look at if you notice coverage going down at any point.